src/containers/audio-rewards-page/components/ClaimRewardButton.tsx

```diff
@@ -229,6 +239,12 @@ const ClaimRewardButton = ({
             throw new Error()
         }
       } else {
+        dispatch(
+          increaseBalance({
+            amount: stringAudioToStringWei(amount.toString() as StringAudio)
+          })
+        )
```

Comment:
rather than duplicating this in two places (we're totally gonna forget this somewhere), we could have rewards sagas listen for the setUserChallengeComplete action and increase the balance there. but your call